packages/cds-lsp/package.yaml

```
@@ -0,0 +1,18 @@
+---
+name: cds-ls
+description: Language server for CDS
+homepage: https://cap.cloud.sap
+licenses:
+  - proprietary
+languages:
+  - CDS
+categories:
+  - LSP
+  # - Formatter
+
+source:
+  id: pkg:npm/@sap/cds-lsp@7.4.0
```

We'll have to encode the `@` sign, otherwise anything right of it will be parsed as the version component, per the PURL spec:

```suggestion
  id: pkg:npm/%40sap/cds-lsp@7.4.0
```